docs/reference/run.md

```diff
@@ -331,6 +331,10 @@ container's namespaces in addition to the `loopback` interface.  An IP
 address will be allocated for containers on the bridge's network and
 traffic will be routed though this bridge to the container.
 
+Service discovery is unavailable on the default bridge network. Containers can
```

Comment:
Same here. I would like to use the word `service discovery` at the most appropriate place. I think we should reiterate the fact that `--link` is mandatory for containers to communicate with each other with name in the `default bridge network`